src/utils/marketxyz/fetchTokenData.ts

```diff
@@ -0,0 +1,59 @@
+import { ChainId } from '@uniswap/sdk';
+import fetch from 'isomorphic-unfetch';
+
+export const ETH_TOKEN_DATA = {
+  symbol: 'ETH',
+  address: '0x0000000000000000000000000000000000000000',
+  name: 'Ethereum Network Token',
+  decimals: 18,
+  color: '#627EEA',
+  overlayTextColor: '#fff',
+  logoURL:
+    'https://icons.iconarchive.com/icons/cjdowner/cryptocurrency-flat/64/Ethereum-ETH-icon.png',
```

Comment:
we should not use url from external servers

Reply:
This is unused and removed